tests/smoketest/smoketest.sh

```diff
@@ -130,17 +127,24 @@ echo "*** [INFO] Logs from alertmanager..."
 oc logs "$(oc get pod -l app=alertmanager -o jsonpath='{.items[0].metadata.name}')" -c alertmanager
 echo
 
-echo "*** [INFO] Cleanup resources..."
-if $CLEANUP; then
-    oc delete "job/stf-smoketest-${NAME}"
-fi
+# moved this check to near the end as previously noted the delivery of this alert and the resulting trap delivery does take some time
+echo "*** [INFO] Check status of SNMP webhook trap delivery"
+SNMP_WEBHOOK_POD=$(oc get pod -l "app=default-snmp-webhook" -ojsonpath='{.items[0].metadata.name}')
+oc logs "$SNMP_WEBHOOK_POD" | grep 'Sending SNMP trap'
+SNMP_WEBHOOK_STATUS=$?
 echo
 
 if [ $SNMP_WEBHOOK_STATUS -ne 0 ]; then
     echo "*** [FAILURE] SNMP Webhook failed"
     exit 1
 fi
 
+echo "*** [INFO] Cleanup resources..."
```

Comment:
Why did this move? The jobs it cleans up aren't related to the webhook pod, and now I believe that if the tests fail for basically any reason that would break SNMP, the script will `exit 1` and not do the cleanup.